app/src/main/java/com/example/inventory/ui/item/ItemEntryScreen.kt

```diff
@@ -91,18 +93,21 @@ fun ItemEntryBody(
     modifier: Modifier = Modifier
 ) {
     Column(
-        modifier = modifier
-            .fillMaxWidth()
-            .padding(16.dp),
-        verticalArrangement = Arrangement.spacedBy(32.dp)
+        modifier = modifier.padding(dimensionResource(id = R.dimen.padding_medium)),
+        verticalArrangement = Arrangement.spacedBy(dimensionResource(id = R.dimen.padding_large))
```

Comment:
is verticalArrangement an optional param?

Reply:
swapped the order